### DISCUSSION TITLE
【a client-side exception】浏览器报错，原因未知

### DISCUSSION THREAD

**Discussion by liano3** [tangly1024/NotionNext#1975] [Q&A]
**描述bug**
文章中含LaTex公式时经常加载错误，报错信息为：Application error: a client-side exception has occurred (see the browser console for more information).

**截图**
![image](https://github.com/tangly1024/NotionNext/assets/100564124/cd6ebbe6-0a69-47be-bc2e-b64f73033e28)

**环境**
- 操作系统: [windows]
- 浏览器 [edge]
- NotionNext版本 [4.0.12]
- 主题 [next](应该与主题无关)

**Comment by liano3** [comment]
发现好像不是Latex公式的问题，示例文章也有公式，但是是正常的，小白看不懂了😢
问题就出在 [动态规划(下)](https://blog.liano.top/article/07683fec-651f-40a6-ae1c-9c13f184caa4) 这几篇

**Comment by Angel0726** [comment]
标记，

**Comment by happyzhangyyds** [comment]
> 问题就出在 [动态规划(下)](https://blog.liano.top/article/07683fec-651f-40a6-ae1c-9c13f184caa4) 这几篇

android打开您这篇文章的时候出现报错。
![Screenshot_2023-08-11-10-09-24-68_a252b927494330cdc2c8ba3b3f952e5e.jpg](https://github.com/tangly1024/NotionNext/assets/119606309/6f011343-84ca-4bfc-bc34-15509a12e1cd)

**Comment by liano3** [comment]
> > 问题就出在 [动态规划(下)](https://blog.liano.top/article/07683fec-651f-40a6-ae1c-9c13f184caa4) 这几篇
> 
> android打开您这篇文章的时候出现报错。 ![Screenshot_2023-08-11-10-09-24-68_a252b927494330cdc2c8ba3b3f952e5e.jpg](https://user-images.githubusercontent.com/119606309/259909160-6f011343-84ca-4bfc-bc34-15509a12e1cd.jpg)

对的，我说的就是这个报错，不知道为什么😢

**Comment by Angel0726** [comment]
你这个问题跟之前提到过的几个问题很像。但是他们没你这么严重

**Comment by tangly1024** [comment]
请尝试更新到最新版本，目前是4.2.0

**Comment by DFBlowing** [comment]
我也出现了上述的问题，并且原因和你很类似，可能是公式代码本身出现了问题，详细的解决方法可以到我的博客里面查看 [博客链接](https://www.xiaoyuezhou.us.kg/error-1) ，应该对大家有帮助

**Reply by GHXY333** [reply]
> 我也出现了上述的问题，并且原因和你很类似，可能是公式代码本身出现了问题，详细的解决方法可以到我的博客里面查看 [博客链接](https://www.xiaoyuezhou.us.kg/error-1) ，应该对大家有帮助

博客链接显示嗯… 无法访问此页面。可以修复吗

**Reply by DFBlowing** [reply]
试试这个 https://blog.xiaoyuezhou.top/error

**Comment by eddiehe99** [comment]
尝试更新到最新版本似乎能解决问题。
